src/Model/ImageFusion.py

```diff
@@ -347,7 +347,7 @@ def get_fused_window(level, window):
     color_sagittal = {}
     color_coronal = {}
 
-    windowing = (int(level), int(window))
+    windowing = (int(level-1024), int(window))
```

Comment:
Please change the "magic number" 1024 here to the constant CT_RESCALE_INTERCEPT

Reply:
Missed this, thanks and done!